index.html

```diff
@@ -3,29 +3,58 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>My Portafolio</title>
     <link rel="stylesheet" href="style.css">
     <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;600;700&display=swap" rel="stylesheet">
 </head>
 <body>
-    <nav>
-        <ul class="flex-nav">
-            <li class="logo">My logo</li>
-            <li class="nav-item">Portafolio</li>
-            <li class="nav-item">About</li>
-            <li class="nav-item">Contact</li>
-        </ul>
-    </nav>
-    <div class="text-box">
-        <h1>I'm Alexandra.</h1>
-        <h1>Glad to see you!</h1>
-        <span>I'm a software developer! I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't heasitate to contact me.</span>
-        <p class="conect">LET'S CONECT</p>
-        <div class="social">
-            <img src="https://cdn1.iconfinder.com/data/icons/social-media-vol-3-2/24/_x-512.png" alt="X">
-            <img src="https://cdn4.iconfinder.com/data/icons/social-icons-16/512/LinkedIn-512.png" alt="linkedin">
-            <img src="https://cdn1.iconfinder.com/data/icons/social-media-circle-7/512/Circled_Medium_svg5-512.png" alt="Medium">
-            <img src="https://cdn1.iconfinder.com/data/icons/logotypes/32/github-512.png" alt="Medium">
+    <div class="Firs-page">

```

Comment:
Small typo here impacting your styles

```suggestion
    <div class="First-page">

```